src/features/island/hud/components/codex/Codex.tsx

```diff
@@ -52,6 +51,17 @@ export const Codex: React.FC<Props> = ({ show, onHide }) => {
     },
   ] = useActor(gameService);
 
+  const {
+    username,
+    bertObsession,
+    npcs,
+    bounties,
+    delivery,
+    choreBoard,
+    kingdomChores,
+    faction,
+  } = state;
```

Comment:
This would be better if you used `useSelector` here for each of these pieces of state.